scripts/multiqc

```diff
@@ -181,6 +187,22 @@ make_data_dir, data_format, zip_data_dir, force, plots_flat, plots_interactive,
     config.plots_force_flat = plots_flat
     config.plots_force_interactive = plots_interactive
     config.kwargs = kwargs # Plugin command line options
+
+    # Add files if --is-file-list option is given
+    if is_file_list:
+        if len(analysis_dir) > 1:
+            raise ValueError("If --is-file-list is giving, analysis_dir should have only one plain text file.")
+        if os.path.isdir(analysis_dir[0]):
+            raise ValueError("If --is-file-list is giving, analysis_dir should be a plain text file.")
+        config.analysis_dir = ()
+        with (open(analysis_dir[0])) as in_handle:
+            for line in in_handle:
+                if os.path.exists(line.strip()):
+                    config.analysis_dir += (os.path.abspath(line.strip()),)
+        if len(config.analysis_dir) == 0:
+            logger.warning("No files were added from %s using --is-file-list option." % analysis_dir[0])
```

Comment:
ups, i forgot to change that. Now you see this: 

[INFO   ]         multiqc : This is MultiQC v0.6.dev0
[WARNING]         multiqc : No files were added from Test1_bcbio.txt using --is-file-list option.
[WARNING]         multiqc : Please, check that Test1_bcbio.txt contains correct file paths.
[WARNING]         multiqc : No analysis results found. Cleaning up..
[INFO   ]         multiqc : MultiQC complete

I added the warning to follow with the logic I saw. Like the last warning was there already, so I thought to add more info.  But yeah, probably should be an error instead.